### PR TITLE
Add Ireland to all-prod 🇮🇪

### PR DESCRIPTION
**Pull request by andrewpbrett**
I think this is everything we need to do - @Matt-Yorkley want to double check?

### CODE REVIEW

**Review by sauloperez — APPROVED** [review]

File: inventory/hosts [hunk 1]
```diff
@@ -182,6 +191,7 @@ es-prod
 fr-prod
 uk-prod
 de-prod
+ie-prod
```
Comment (Matt-Yorkley) [inline inventory/hosts:194]: :+1:

File: inventory/hosts [hunk 2]
```diff
@@ -111,6 +111,15 @@ uk-prod
...
 uk-staging2
 
+#------------------------------------------------------------------------------
+# Ireland
+
+[ie-prod]
+openfoodnetwork.ie ansible_host=167.172.44.121
+
+[ie:children]
+ie-prod
```
Comment (Matt-Yorkley) [inline inventory/hosts:121]:
```suggestion
```

**Comment by sauloperez** [comment]
I went ahed commit your suggestion @Matt-Yorkley so we merge it.

**Comment by sauloperez** [comment]
I think it'd be about time we split this long hosts file into a file per instance, don't you agree?

**Pull request merged**